### DISCUSSION TITLE
Where can I see error log?

### DISCUSSION THREAD

**Discussion by shibukawa** [microsoft/wslg#121] [Q&A]
I installed Insider Preview on two computers and tried WSLg. One works perfectly, but another one doesn't.

Can I see any logs to research solution? I checked Windows' Event Viewer and wsl --system. But I couldn't find anything.

**Comment by onomatopellan** [comment]
There is the `/mnt/wslg/weston.log` file. Also the wiki is a good start for diagnosing WSLg issues:
https://github.com/microsoft/wslg/wiki/Diagnosing-%22cannot-open-display%22-type-issues-with-WSLg

**Reply by shibukawa** [reply]
Thank you!
